crates/worker/src/partition/mod.rs

```diff
@@ -253,13 +262,21 @@ where
                 FindTailAttributes::default(),
             )
             .await?;
-        info!(
+        debug!(
+           partition_id = %self.partition_id,
```

Comment:
Well. I'm a little torn on how we should log context within partition processor. At the moment, we set `partition_id` in the parent span (see line 248), so you don't need to add partition_id to every log message. That said, the fact that we have a single ever-running span makes distributed traces look weird.

I'm happy either way, but we probably want to research this a bit and figure out a strategy. The good news is that tracing is now decoupled from user-service tracing so we can do what we like.

Reply:
I reduced some of these as you can infer the partition id from the snapshot directory path anyway. I don't normally look at the traces - I should get into the habit - but I liked seeing the explicit partition id in console logs. 

No opinion on structuring the tracing overall yet 😅